includes/constants/class-fraud-outcome-status.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Class Fraud_Outcome_Status
+ *
+ * @package WooCommerce\Payments
+ */
+
+namespace WCPay\Constants;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * This class gives a list of all the possible fraud outcome status constants.
+ *
+ * @psalm-immutable
+ */
+class Fraud_Outcome_Status extends Base_Constant {
+	const ALLOW          = 'allow';
```

Comment:
I think these statuses mostly match the statuses in https://github.com/Automattic/woocommerce-payments/blob/ee28c1128308c4d0b71a1b7199bbb845379a6c1f/includes/fraud-prevention/models/class-rule.php#L16-L24, except the review results. Do you think it'll be useful for being future-proof if we reference at least `allow`, `review`, and `block` ones to the `Rule` constants?

Reply:
Do you mean setting the constants in `Fraud_Outcome_Status` to something like:

```
const ALLOW = Rule::FRAUD_OUTCOME_ALLOW
```

Reply:
This was updated in https://github.com/Automattic/woocommerce-payments/pull/5782/commits/9f94c3d64200f8ccfd5eba5bc510472af6ff7b36